previews.md

```diff
@@ -1,13 +1,48 @@
 # AKS Preview Features and Related Projects
 
-At any given time, there can be multiple early stage features available in AKS behind a *feature flag*, along with a set of related projects available elsewhere on GitHub that you may wish deploy manually on the service. 
+At any given time, there can be multiple early stage features available in AKS behind a *feature flag*, along with a set of related projects available elsewhere on GitHub that you may wish deploy manually on the service.
 
 In most cases, these features and associated projects will eventually make their way into AKS, or at least be supported as 1st class extensions. But before they get there, we need sufficient usage from early adopters to validate their usefulness and quality.
 
 The purpose of this page is to capture these features and associated projects in a single place.
 
+**Note**: AKS Preview features are self-service, opt-in. They are provided to gather feedback and bugs from our community. However, they are not supported by Azure technical support and adding these to existing clusters will render the clusters as unsupported until the feature exits preview and graduate to general availability. If you encounter issues with preview features, please file issues on this GitHub repo with the name of the preview feature in the bug title.
+
+## Getting Started
+
+In order to use / opt into preview features, you will need to use the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) and ensure it is up to date with the latest release. Once that is complete, you must install the `aks-preview` extension via:
+
+```
+az extension add --name aks-preview
+```
+
+**Warning**: Installing the preview extension will update the CLI to use the **latest** extensions and API. Please do not enable this for production clusters. In order to uninstall the extension, you can do the following:
+
+```
+az extension remove --name aks-preview
+```
+
+
 ## Preview features
 
+### Virtual Machine Scale Sets (VMSS)
+
+
+You can opt into the preview by registering the feature flag:
+
+```
+az feature register -n VMSSPreview --namespace Microsoft.ContainerService
+```
+
+Then refresh your registration of the AKS resource provider:
+
+```
+az provider register -n Microsoft.ContainerService
+```
+
+To create a cluster with network policy enabled, use the `--network-policy` switch in `az aks create` and specify the network policy plugin you would like to use, `azure` or `calico`. Note that both require the `azure` network plugin.
```

Comment:
Should this be under the below header "Kubernetes Network Policy"? Seems odd its under the VMSS header.

Reply:
sigh, copy paste error